src/frontend/src/utils/condition.rs

```diff
@@ -239,6 +240,77 @@ impl Condition {
         .unwrap()
     }
 
+    /// generate range scans from each arm of `OR` clause and merge them
+    /// Currently, only support equal type range scans.
+    /// Keep in mind that range scans can not be overlap, otherwise duplicate rows will occur.
```

Comment:
```suggestion
    /// Generate range scans from each arm of `OR` clause and merge them.
    /// Currently, only support equal type range scans.
    /// Keep in mind that range scans can not overlap, otherwise duplicate rows will occur.
```